test.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
If it is a test file, please put it under the tests folder with certain filesystem structures like how EDA did.